notebooks/COS/ExceptionReport/ExceptionReport.ipynb

```diff
@@ -0,0 +1,1448 @@
+{
```

Comment:
<p>There is some code duplication in plotting the same data tice for flux. vs wavelength. You can wrap the plotting logic into a loop to reduce redundancy.</p><p> </p>

<pre class="ql-syntax" spellcheck="false">for i, axis in enumerate(ax[0]):
&nbsp; &nbsp; # Plotting flux vs wavelength on top row
&nbsp; &nbsp; axis.plot(wl, flux,
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; lw=2,
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; color="black")
&nbsp; &nbsp; axis.set_xlabel(r'Wavelength [$\AA$]')
&nbsp; &nbsp; axis.set_ylabel(r'Flux [$erg\ s^{-1}\ cm^{-2}\ \AA^{-1}$]')
&nbsp; &nbsp; if i == 0:
&nbsp; &nbsp; &nbsp; &nbsp; axis.set_title("Flux vs Wavelength")
&nbsp; &nbsp; else:
&nbsp; &nbsp; &nbsp; &nbsp; axis.set_title("Flux vs Wavelength, ZOOMED")
&nbsp; &nbsp; &nbsp; &nbsp; axis.set_ylim(0, 0.6e-13)


</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/305/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
@srosagomez, this feedback is from an engineering perspective. However, if you prefer to keep your tutorial as is for clarity, that's fine with me. A clear workflow is more important than avoiding code duplication in the tutorial. Please check the readability and let me know :)

Reply:
<p>I noticed that you have already implemented the code in the above cell. Could you please do the same for this code cell?</p>